src/test/java/teammates/test/cases/ui/browsertests/InstructorFeedbackEditPageUiTest.java

```diff
@@ -502,6 +503,127 @@ private void testCopyQuestion() throws Exception {
         
     }
     
+    private void testChangeFeedbackGiver() {
+        feedbackEditPage = getFeedbackEditPage();
+        feedbackEditPage.clickQuestionEditForQuestion1();
+        
+        ______TS("change giver to \"Students in this course\" from \"Me (Session creator)\"");
+        feedbackEditPage.selectGiverTypeForQuestion1("Students in this course");
+        assertTrue(feedbackEditPage.isAllRecipientOptionsDisplayed(1));
+        
+        ______TS("change giver to \"Instructors in this course\" from \"Students in this course\"");
+        feedbackEditPage.selectGiverTypeForQuestion1("Instructors in this course");
+        assertTrue(feedbackEditPage.isRecipientOptionDisplayed(
+                            FeedbackParticipantType.SELF, 1));
+        assertTrue(feedbackEditPage.isRecipientOptionDisplayed(
+                            FeedbackParticipantType.STUDENTS, 1));
+        assertTrue(feedbackEditPage.isRecipientOptionDisplayed(
+                            FeedbackParticipantType.INSTRUCTORS, 1));
+        assertTrue(feedbackEditPage.isRecipientOptionDisplayed(
+                            FeedbackParticipantType.TEAMS, 1));
+        assertTrue(feedbackEditPage.isRecipientOptionDisplayed(
+                            FeedbackParticipantType.OWN_TEAM, 1));
+        assertFalse(feedbackEditPage.isRecipientOptionDisplayed(
+                            FeedbackParticipantType.OWN_TEAM_MEMBERS, 1));
+        assertFalse(feedbackEditPage.isRecipientOptionDisplayed(
+                            FeedbackParticipantType.OWN_TEAM_MEMBERS_INCLUDING_SELF, 1));
+        assertTrue(feedbackEditPage.isRecipientOptionDisplayed(
+                            FeedbackParticipantType.NONE, 1));
```

Comment:
Most of them don't need line breaks. I think only the `OWN_TEAM_MEMBERS_INCLUDING_SELF` needs one.